.eslintrc.js

```diff
@@ -13,11 +13,22 @@ module.exports = {
       "prettier/prettier": "error"
     },
     overrides: [
+      {
+        files: ['**/*.ts'],
+        parser: '@typescript-eslint/parser',
+        plugins: ['prettier', '@typescript-eslint'],
+        extends: [
+          'eslint:recommended',
+          'plugin:@typescript-eslint/recommended',
+          'prettier',
+          'prettier/@typescript-eslint',
+        ],
+      },
```

Comment:
I don't think you have to duplicate the plugins and whatnot. You should be able to just add what you need:

```js
{
  files: ['**/*.ts'],
  parser: '@typescript-eslint/parser',
  plugins: ['@typescript-eslint'],
  extends: [
    'plugin:@typescript-eslint/recommended',
    'prettier/@typescript-eslint',
  ]
}
```

Reply:
how would you then specify if you didn't want certain plugins for particular files?

Reply:
I don't know, but I _am_ pretty sure that this merges with the config from above (you can test locally if you want). Do actually have that need?